core/src/main/java/org/apache/iceberg/rest/responses/ErrorResponseParser.java

```diff
@@ -64,6 +66,13 @@ public static void toJson(ErrorResponse errorResponse, JsonGenerator generator)
     generator.writeStringField(MESSAGE, errorResponse.message());
     generator.writeStringField(TYPE, errorResponse.type());
     generator.writeNumberField(CODE, errorResponse.code());
+    if (errorResponse.stack() != null) {
+      generator.writeArrayFieldStart(STACK);
+      for (String line : errorResponse.stack()) {
+        generator.writeString(line);
+      }
+      generator.writeEndArray();
```

Comment:
Nit: Should we consider writing the `stack` field even if it's `null`?

Reply:
I think it is good this way. Why send extra information that is already clear?